backend/api/v2beta1/error.proto

```diff
@@ -15,7 +15,7 @@
 syntax = "proto3";
 
 option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
-package v2beta1;
+package kubeflow.pipelines.backend.api.v2beta1;
```

Comment:
As we decided to use `google.rpc.status.proto`, `error.proto` can actually be removed.